client/src/components/PinnedFAQ/PinnedFAQ.js

```diff
@@ -26,47 +26,59 @@ export class PinnedFAQ extends React.Component {
       TM,
       is_initially_expanded,
       background_color,
+      no_pin,
```

Comment:
seems unintuitive to have a `no_pin` option for a component called "PinnedFAQ" but it doesn't make sense to have a pin option inside the form since I can't pin things inside the modal